includes/optional-modules/class-media-partners.php

```diff
@@ -47,6 +47,7 @@ public static function switch_from_standalone_plugin() {
 			deactivate_plugins( 'newspack-media-partners/newspack-media-partners.php' );
 			Settings::activate_optional_module( 'media-partners' );
 		}
+		return true;
```

Comment:
just curious. Why we need this?

Reply:
We don't, I've accidentally used filters instead of actions and then mindlessly followed linter's advice :D Good that you asked! Fixed in ed47d6f808645a22d6636e61ac0297af0918258f

Reply:
ah, I didn't even notice! perfect